### DISCUSSION TITLE
cannot run command of piccolo in anaconda env

### DISCUSSION THREAD

**Discussion by shahensaifullah** [piccolo-orm/piccolo#574] [Q&A] [answered]
> version: piccolo==0.82.0

when i run comamnd **piccolo migrations new my_app**, it shows a error
`'piccolo' is not recognized as an internal or external command,`
other command is: **python -m piccolo migrations new sql_app --auto**.
error: 
`C:\ProgramData\Anaconda3\envs\bongoSheepNuxt\python.exe: No module named piccolo.__main__; 'piccolo' is a package and cannot be directly executed`
how can I run piccolo command in anaconda env?
or is there any any to run piccolo in __name__ function?
here is mine:
```
if __name__ == "__main__":
    uvicorn.run(app, host="127.0.0.1", port=8000, log_level="debug")
```

**Comment by dantownsend** [comment, marked as answer]
We have this line in `setup.py` which should make the piccolo CLI available:

https://github.com/piccolo-orm/piccolo/blob/0c44bb22be451e27318ddc2594e7e5a775494f80/setup.py#L96

I know that Anaconda uses a different packaging system, so it must be ignoring `setup.py`. It's something we need to investigate.

You can try this work around instead:

```
python -m piccolo.main migrations new sql_app --auto
```

Note how it's `piccolo.main`.

**Reply by shahensaifullah** [reply]
Alright, it is working now.. thanks.
